channel/handlers.go

```diff
@@ -2994,33 +2994,49 @@ func (server Server) handleChatEvent(conn mnet.Server, reader mpacket.Reader) {
 	}
 }
 
-func (server *Server) getAffectedPartyMembers(p *party, src *player, includeSelf bool) []*player {
+func (server *Server) getAffectedPartyMembers(p *party, src *player, affected byte) []*player {
```

Comment:
I think this can just be a function not a method as I don't see where this function reads/writes to server state?